uw-frame-components/portal/misc/partials/add-to-home.html

```diff
@@ -1,42 +1,37 @@
-<!-- ADD MORE TO HOME BUTTON -->
-<div ng-if='actionLinkUrl && !addToHome'>
-  <md-button class="md-primary link-div" ng-href="{{actionLinkUrl}}" hide-sm hide-xs>
-    <span><i class="fa fa-fw {{actionLinkIcon}}"></i> {{actionLinkText}}</span>
-  </md-button>
-  <md-menu-item hide-gt-sm>
-    <md-button class="md-primary link-div" ng-href="{{actionLinkUrl}}">
-      <span><i class="fa fa-fw {{actionLinkIcon}}"></i> {{actionLinkText}}</span>
-    </md-button>
-  </md-menu-item>
-</div>
-
 <!-- ADD -THIS- TO HOME BUTTON -->
-<div ng-controller="AddToHomeController" ng-if="addToHome">
+<div ng-controller="AddToHomeController">
   <!-- Button for medium+ screens -->
-  <md-button class="md-default link-div" aria-label="add to home" hide-xs hide-sm ng-hide='inHome' ng-click='addToHome()'>
+  <md-button class="md-default link-div" aria-label="add to home" hide-xs hide-sm ng-hide="inHome" ng-click="addToHome()">
     <span ng-if="!successfullyAdded || !addToHomeFailed">
-      <i class="fa fa-fw {{actionLinkIcon}}" ng-class="{ 'fa-spin' : savingAddToHome, 'fa-plus' : !actionLinkIcon}"></i> {{actionLinkText}}
+      <md-icon ng-hide="savingAddToHome">add</md-icon>
+      <i class="fa fa-fw fa-spin" ng-show="savingAddToHome"></i>
+      Add to home
     </span>
-    <span ng-show='successfullyAdded'>
-      <i class="fa fa-fw fa-check-square"></i> {{actionLinkText}} Saved
+    <span ng-show="successfullyAdded">
+      <md-icon>check</md-icon> Added to home
     </span>
-    <span ng-show='addToHomeFailed'>
-      <i class="fa fa-fw fa-exclamation-triangle"></i> {{actionLinkText}} Failed
+    <span ng-show="addToHomeFailed">
+      <md-icon>error</md-icon> Add to home failed
     </span>
   </md-button>
 
   <!-- Button for small and xs screens -->
-  <md-menu-item hide-gt-sm>
-    <md-button class="md-default link-div" aria-label="add to home" ng-click="addToHome()" ng-hide="inHome">
+  <md-button class="md-default link-div" aria-label="add to home" hide-gt-sm ng-hide="inHome" ng-click="addToHome()">
     <span ng-if="!successfullyAdded || !addToHomeFailed">
-      <i class="fa fa-fw {{actionLinkIcon}}" ng-class="{ 'fa-spin' : savingAddToHome, 'fa-plus' : !actionLinkIcon}"></i> {{actionLinkText}}
+      <md-icon ng-hide="savingAddToHome">add</md-icon>
+      <i class="fa fa-fw fa-spin" ng-show="savingAddToHome"></i>
+      <span class="aria-only">Add to home</span>
```

Comment:
1. Would it be better to `aria-label the `<i class="fa fa-fw fa-spin"` than to introduce this `<span` with its `aria-only` class?
2. I'm not sure "Add to home" is the best text to use here.  The button is already `aria-label`ed "add to home" above, so I think labeling the control before actuation is covered. But the visual-only `<i class="fa fa-fw fa-spin"`, that's doing the work of indicating to the user an in-progress operation. So maybe if it had an `aria-label` of "Saving addition to home screen" or so that would give it an equivalent text representation of the indicator.